server/src/main/java/org/elasticsearch/indices/analysis/AnalysisModule.java

```diff
@@ -134,28 +128,6 @@ private static NamedRegistry<AnalysisProvider<TokenFilterFactory>> setupTokenFil
     ) {
         NamedRegistry<AnalysisProvider<TokenFilterFactory>> tokenFilters = new NamedRegistry<>("token_filter");
         tokenFilters.register("stop", StopTokenFilterFactory::new);
-        // Add "standard" for old indices (bwc)
-        tokenFilters.register("standard", new AnalysisProvider<TokenFilterFactory>() {
```

Comment:
OK, talking with @javanna we might have an issue with archive indices....we will discuss to find the best way forward here to clean up the core analysis module.

Reply:
@javanna I changed some tests locally and indeed, adding a standard filter causes opening an index to fail with:

> » [2024-12-18T16:20:46,061][WARN ][o.e.s.RestoreService     ] [5-6-16-1] could not import mappings for legacy index winlogbeat java.lang.IllegalArgumentException: Custom Analyzer [my_custom_analyzer] failed to find filter under name [standard]

:(

Is there a way to add the standard tokenizer via the plugin? That way we remove it from the main server module?